src/hooks/useLock.ts

```diff
@@ -50,7 +50,7 @@ export const useLock = () => {
   }
 
   const lockIQ = async (amount: number, lockPeriod: number) => {
-    const parsedAmount = ethers.utils.parseEther(amount.toString())
+    const parsedAmount = ethers.utils.parseEther(`${amount}`)
```

Comment:
```suggestion
    const parsedAmount = ethers.utils.parseEther(amount)
```

Reply:
@kesar parseEther takes a string as its parameter.

Reply:
oh yeah, good point 👍🏻 . but the problem is that I think we talked to use bigInts, and here we are sending number. ideally it should expect bigInt and use formatEther to display proper value